### PR TITLE
EIP5006: ERC-1155 Usage Rights Extension

### PR DESCRIPTION
**Pull request by emojidao**
This standard is an extension of ERC-1155. It proposes an additional role (user) which can be granted to addresses. The user role represents permission to "use" the NFT, but not be able to transfer it or set operators.

### CODE REVIEW

**Comment by Hbjjhbk** [comment]
Spoko

**Comment by Hbjjhbk** [comment]
Dzięki

**Comment by Hbjjhbk** [comment]
Ok

File: EIPS/eip-5006.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+}
+```
+
+## Security Considerations
+
+This EIP standard can completely protect the rights of the owner, the owner can change the NFT user.
+
+## Copyright
+Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+
```
Comment (lightclient) [inline EIPS/eip-5006.md:343]:
```suggestion
```

**Review by lightclient — CHANGES_REQUESTED** [review]

**Review by lightclient — APPROVED** [review]

**Review by 0xanders — APPROVED** [review]

**Pull request merged**